### ISSUE TITLE
Expose glasbey variants exposed by the (python) colorcet package

### ISSUE THREAD

**Issue by anntzer** [tlambert03/cmap#18]
* cmap version: 0.0.3
* Python version: any
* Operating System: any

### Description

colorcet (the Python package) exposes a few variants on the glasbey color series (not really colormaps, but rather lists of maximally different colors), per https://github.com/holoviz/colorcet/issues/11#issuecomment-478755134.

Admittedly these are constructs from the colorcet maintainers and not so much from the original glasbey paper, but it would be nice to include them, as I don't think there's any other color lists (designed for that use case) in cmap.

(Also, as a side point, glasbey should probably be categorized as "qualitative", not as "misc.")

### What I Did

N/A

**Comment by tlambert03** [comment]
the reason I haven't done that (yet) is because I added the complete functional form here: https://github.com/tlambert03/cmap/tree/main/src/cmap/data/glasbey ...  So, I'd kinda like to try to "unify" the glasbey variants here and expose parametrized functional colormap variants (https://github.com/tlambert03/cmap/issues/19)

I'm not opposed to exposing the "preselected" variants from the colorcet lib (particularly as a "stop gap") ... but it'd be nicer in the long term to expose them under the glasbey namespace, as parameters to the function above.

> glasbey should probably be categorized as "qualitative", not as "misc."

I can see this one going both ways.  Wanna make a strong(er) argument for "qualitative"?

**Comment by anntzer** [comment]
> I'm not opposed to exposing the "preselected" variants from the colorcet lib (particularly as a "stop gap") ... but it'd be nicer in the long term to expose them under the glasbey namespace, as parameters to the function above.

Sure, both seem reasonable.

> Wanna make a strong(er) argument for "qualitative"?

The original paper title (Colour Displays for Categorical Images) clearly states these are intended for categorical images, which I think is what you mean by "qualitative"?

**Comment by tlambert03** [comment]
yeah, I mostly went with the [matplotlib categories](https://matplotlib.org/stable/tutorials/colors/colormaps.html) ... and I agree with you after having another look.  happy to make that change.

as for exposing more glasbey variants.  i guess we could just include the pre-built ones used by colorcet.  I know @kevinyamauchi was also looking to have some pre-built variants exposed as well.
